test/runtests.jl

```diff
@@ -36,7 +36,7 @@ include("util.jl")
     @testset "StaticMH" begin
         # Set up our sampler with initial parameters.
         spl1 = StaticMH([Normal(0,1), Normal(0, 1)])
-        spl2 = StaticMH(MvNormal(zeros(2), I))
+        spl2 = StaticMH(2)
```

Comment:
But now we no longer have tests for the `StaticMH(MvNormal(zeros(2), I))` scenario, right?

Please add an additiional sampler `spl3` instead.

Reply:
done!